.github/workflows/ci.yaml

```diff
@@ -2,7 +2,7 @@
 name: CI
 
 env:
-  VERSION_GO: '1.19.8'
+  VERSION_GO: '1.21.5'
```

Comment:
We'd better keep it aligned with the latest helm version that helm-diff supports! It should be the latest of 1.20 this time then?

https://github.com/helm/helm/blob/c8b948945e52abba22ff885446a1486cb5fd3474/.github/workflows/release.yml#L28

Reply:
switched to 1.20.12